services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -114,7 +114,7 @@ async def docker_compose_up(
     result = await _write_file_and_spawn_process(
         compose_spec_yaml,
         command=f'docker-compose {_docker_compose_options_from_settings(settings)} --project-name {settings.DYNAMIC_SIDECAR_COMPOSE_NAMESPACE} --file "{{file_path}}" up'
-        " --no-build --detach",
+        " --no-build --no-start",
```

Comment:
Q? Where does these services start then?

Reply:
A: there is a call to docker-compose start coming right after that one... yes I agree it needs some more love, and I think I will rename that fct

Reply:
aha ... now i see. yes, it is better called create then.